modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-1_{context}"]
+= Release notes for Red Hat {pipelines-title} Technology Preview 1.1
+
+[id="new-features-1-1_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.1 is now available on {product-title} {product-version}. {pipelines-title} TP 1.1 is updated to support:
+
+* Tekton Pipelines 0.14.3
+* Tekton `tkn` CLI 0.11.0
+* Tekton Triggers 0.6.1
+* ClusterTasks based on Tekton Catalog 0.14
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.1.
+
+=== Pipelines
+* Workspace support for VolumeClaimTemplates has been added:
+** The VolumeClaimTemplate for a PipelineRun and TaskRun can now be added as a volume source for Workspaces. The tekton-controller then creates a PersistentVolumeClaim (PVC) using the template that is seen as a PVC for all TaskRuns in the Pipeline. Thus you do not need to define the PVC configuration every time it binds a workspace that spans multiple tasks.
+** Support to find the name of the PersistentVolumeClaim when a VolumeClaimTemplate is used as a volume source is now added using variable substitution.
```

Comment:
There are a lot of instances of "is now added" that don't sound right. You're saying "is now" present tense with "added" past tense. I would update these to either say something like "is now available" or "has been added". 

Multiple instances throughout PR - I won't comment for each.

Reply:
Fixed